web/ViewModels/LandlordProfileModel.cs

```diff
@@ -29,12 +29,9 @@ public class LandlordProfileModel
     [DisplayName("Company")]
     public string? CompanyName { get; set; }
 
-    [Required]
-    [EmailAddress]
-    public string Email { get; set; } = string.Empty;
+    [Required] [EmailAddress] public string Email { get; set; } = string.Empty;

```

Comment:
I've changed the rules surrounding this now:
![image](https://user-images.githubusercontent.com/36637730/182351232-85b3e9f4-5f97-4b6d-9f8e-dd5a1e5b19e3.png)
It enforces them to be written as before

Reply:
Also, when I re-export the settings file it adds in all the other bits for other file extensions, but they only seem to be indent size/style which are in everyone's editors anyway, so it may be easier to leave it in when we make updates.
I've removed them from this update though.

Reply:
Oh sure, if it's regenerating it then fine to leave it in